aspnetcore/mvc/models/file-uploads/sample/FileUploadSample/Filters/DisableFormValueModelBindingAttribute.cs

```diff
@@ -1,36 +1,21 @@
-﻿using System;
-using System.Linq;
 using Microsoft.AspNetCore.Mvc.Filters;
 using Microsoft.AspNetCore.Mvc.ModelBinding;
+using System;
 
 namespace FileUploadSample.Filters
 {
-    #region snippet1
```

Comment:
Line 159 of the topic pulls in this section of the code for display, so the region/endregion should stay.

Reply:
Restored the region https://github.com/aspnet/Docs/pull/5073/commits/72edb9f3249d68531809f849f03112685a249931.

Reply:
Thanks @weitzhandler. Shortening this is wonderful. There's one more tiny nit .... namespace import ordering. We put the `System` namespaces first following what the engineering teams do. Can you move `using System;` to the top?

Reply:
@guardrex Anything that will make you happy man https://github.com/aspnet/Docs/pull/5073/commits/44b0f2c723c02f045baa6ccfa38cbd22b6316d77 😄